SQL/beestation_schema.sql

```diff
@@ -568,6 +568,14 @@ CREATE TABLE `achievement_metadata` (
 	PRIMARY KEY (`achievement_key`)
 ) ENGINE=InnoDB;
 
+DROP TABLE IF EXISTS `metacoin_item_purchases`;
```

Comment:
```suggestion
DROP TABLE IF EXISTS `SS13_metacoin_item_purchases`;
```

Edits to the BeeStation schema need to have the `SS13_` prefix.